kolibri/plugins/learn/assets/src/views/ExamPage/AnswerHistory.vue

```diff
@@ -91,7 +91,7 @@
       },
     },
     $trs: {
-      question: 'Question { num }',
+      question: 'Question { num, number, integer}',
```

Comment:
```suggestion
      question: 'Question { num, number, integer }',
```

Reply:
Did you modify the locale JSON file directly? I see changes to other messages like
```
  "ExamPage.question": "Câu hỏi {num, number, integer} trong số {total, number, integer}",
```

but don't see `ExamPage.vue` in this diff. Same with `ChannelVersion`

Reply:
Thank you @jonboiser , and sorry about that. I will set the fixes only on `.vue` components and update the PR